docker/docker-compose.yml

```diff
@@ -142,7 +118,7 @@ services:
       POSTGRES_DB: "conseil-local"
       POSTGRES_INITDB_ARGS: "--lc-collate=en_US.UTF-8 -E UTF8"
     volumes:
-      - "conseil-sql:/docker-entrypoint-initdb.d/"
+      - "../sql:/docker-entrypoint-initdb.d/"
```

Comment:
You should be able to remove the following lines (L75/L50) above:

```
volumes:
      - "conseil-sql:/root/sql/"
```

and the corresponding volume entry on L125:

```
volumes:
  conseil-sql:
```

Reply:
Done.